src/modules/dashboard/components/TokenMintDialog/TokenMintDialog.tsx

```diff
@@ -61,8 +64,27 @@ const TokenMintDialog = ({
   callStep,
   prevStep,
 }: Props) => {
+  const [isForceOn, setIsForceOn] = useState(false);
   const history = useHistory();
 
+  const { data } = useColonyExtensionsQuery({
+    variables: { address: colonyAddress },
+  });
+
+  const hasVotingExtension = useHasVotingExtension(data);
```

Comment:
This is not scalable. I created something similar - useEnabledExtensions hook. We should have hook which handle all cases, not for each extension one hook. So if my hook doesn't work for you (or has bug), just refactor it to make it more useful across the app

Reply:
Yeah, already saw and changed - there is already an updated version